contracts/Lens/ComptrollerLens.sol

```diff
@@ -23,20 +23,34 @@ contract ComptrollerLens is ComptrollerLensInterface, ComptrollerErrorReporter,
      *  whereas `borrowBalance` is the amount of underlying that the account has borrowed.
      */
     struct AccountLiquidityLocalVars {
-        uint totalCollateral;
-        uint weightedCollateral;
-        uint borrows;
-        uint effects;
-        uint vTokenBalance;
-        uint borrowBalance;
-        uint exchangeRateMantissa;
-        uint oraclePriceMantissa;
-        uint liquidity;
-        uint shortfall;
-        uint256 averageLT; // Average liquidation threshold of all assets in the snapshot
-        uint256 healthFactor; // Health factor of the account, calculated as (weightedCollateral / borrows)
-        uint256 healthFactorThreshold; // Health factor threshold for liquidation, calculated as (averageLT * (1e18 + LiquidationIncentiveAvg) / 1e18)
-        uint256 liquidationIncentiveAvg; // Average liquidation incentive of all assets in the snapshot
+        // Total collateral value supplied by the account (USD, scaled by 1e18)
+        uint256 totalCollateral;
+        // Collateral value weighted by each asset's liquidation threshold (USD, scaled by 1e18)
```

Comment:
I think it can also be weighted by the collateral factors, depending on the use case

Reply:
We can add it if needed, but I think currently, we will be doing all calculations related to liquidations for the `liquidation threshold`.

Reply:
I think we should use `getCollateralFactor` in the following flows:
* borrow
* redeem

And `getLiquidationThreshold` in the following flows:
* liquidation

That is what we do in the Isolated Pools [here](https://github.com/VenusProtocol/isolated-pools/blob/develop/contracts/Comptroller.sol)